providers/keycloak/pyproject.toml

```diff
@@ -25,7 +25,7 @@ build-backend = "flit_core.buildapi"
 
 [project]
 name = "apache-airflow-providers-keycloak"
-version = "0.0.1"
```

Comment:
As mentioned in Slack, should we release it as 1.0.0? Implementation and doc is completed on our side. @bugraoz93